nipype/interfaces/mrtrix3/connectivity.py

```diff
@@ -208,17 +209,15 @@ def _parse_inputs(self, skip=None):
             skip = []
 
         if not isdefined(self.inputs.in_config):
-            from distutils.spawn import find_executable
-
-            path = find_executable(self._cmd)
+            path = shutil.which(self._cmd)
             if path is None:
                 path = os.getenv(MRTRIX3_HOME, "/opt/mrtrix3")
             else:
                 path = op.dirname(op.dirname(path))
 
             self.inputs.in_config = op.join(
                 path,
-                "src/dwi/tractography/connectomics/" "example_configs/fs_default.txt",
+                "src/dwi/tractography/connectomics/example_configs/fs_default.txt",
```

Comment:
shouldn't this be path expanded?
```suggestion
                "src",
                "dwi",
                "tractography",
                "connectomics",
                "example_configs",
                "fs_default.txt",
```

Reply:
It can be, but I don't see much advantage to it. Windows interprets forward slashes as path separators, and this fits on one line.